Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean

```diff
@@ -405,6 +405,17 @@ lemma rnDeriv_le_one_of_le (hμν : μ ≤ ν) [SigmaFinite ν] : μ.rnDeriv ν
   simp only [Pi.one_apply, MeasureTheory.setLIntegral_one]
   exact (Measure.setLIntegral_rnDeriv_le s).trans (hμν s)
 
+lemma rnDeriv_le_one_iff_le [SigmaFinite μ] [SigmaFinite ν] (hμν : μ ≪ ν) :
```

Comment:
Does it work if you replace the assumption by this?
```suggestion
lemma rnDeriv_le_one_iff_le [μ.HaveLebesgueDecomposition ν] (hμν : μ ≪ ν) :
```

Reply:
It does not work, since `rnDeriv_le_one_of_le` and `withDensity_apply'` need `[SigmaFinite ν]`. 
`[μ.HaveLebesgueDecomposition ν] [SigmaFinite ν]` works, should we leave like it is, or use these hypotheses?

Reply:
It does not work, since `rnDeriv_le_one_of_le` and `withDensity_apply'` need `[SigmaFinite ν]`. 
`[μ.HaveLebesgueDecomposition ν] [SigmaFinite ν]` works, should we leave like it is, or use these hypotheses?

Reply:
Ok, I generalized the hypotheses of the other lemmas in the file, where possible.